### PR TITLE
Handle popover showing from side only in sidebar screen

### PR DESCRIPTION
**Pull request by AlfredoAlc**
### Details

The changes made are to support showing from the side the `popover` (FAB options) on iPad.

### Fixed Issues

$ https://github.com/Expensify/App/pull/6278
$ https://github.com/Expensify/App/issues/6520
$ https://github.com/Expensify/App/issues/6519
$ https://github.com/Expensify/App/issues/6522

### Tests

1. Press FAB, options should appear from bottom(iPhone), options should appear from side(iPad).
2. Enter a conversation, press the `+` icon and press `Add attachment`, the next `popover` should appear from bottom.
3. Press emoticons button, `popover` should appear from the bottom and centered from the sides on the screen.
4. Navigate to settings, press avatar to edit profile.
5. Press avatar to change image.
6. Press `Upload photo`, next `popover` should appear from bottom.

### QA Steps

1. Press FAB, options should appear from bottom(iPhone), options should appear from side(iPad).
2. Enter a conversation, press the `+` icon and press `Add attachment`, the next `popover` should appear from bottom.
3. Press emoticons button, `popover` should appear from the bottom and centered from the sides on the screen.
4. Navigate to settings, press avatar to edit profile.
5. Press avatar to change image.
6. Press `Upload photo`, next `popover` should appear from bottom.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### iPhone

https://user-images.githubusercontent.com/53409604/145513505-1c4f1616-b48f-4a07-9d0a-9e5d569005d4.mp4

#### iPad

https://user-images.githubusercontent.com/53409604/145513652-7bb92bea-8a09-46d4-a2b2-7128144cf57d.mp4

#### Android
https://user-images.githubusercontent.com/53409604/145513692-deba593a-886b-48fd-ae3d-c018d7756e56.mov

#### Web

https://user-images.githubusercontent.com/53409604/145513768-4f63e112-6d99-4055-8286-cd60718737c1.mov

### CODE REVIEW

File: src/components/Popover/index.native.js [hunk 1]
```diff
@@ -13,20 +13,17 @@ const propTypes = {
...
-};
+const Popover = props => (
+    <Modal
+        type={props.fromSidebarMediumScreen ? CONST.MODAL.MODAL_TYPE.POPOVER : CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED}
+        popoverAnchorPosition={props.isSmallScreenWidth ? undefined : props.anchorPosition}
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        fullscreen={props.isSmallScreenWidth ? true : props.fullscreen}
+        animationIn={props.isSmallScreenWidth ? undefined : props.animationIn}
+        animationOut={props.isSmallScreenWidth ? undefined : props.animationOut}
```
Comment (Jag96) [inline src/components/Popover/index.native.js:24]: Any reason we're forcing no animation on small screens? We should be keeping the default we currently have for native (iOS/Android) where it slides up from the bottom

**Review by Jag96 — CHANGES_REQUESTED** [review]
It looks like this PR has some regressions, mainly the animations on iOS/Android are now missing (create menu and change profile picture as examples). Can you update this and also add screenshots for the other platforms (iOS non-iPad, Android, Web) to confirm there are no additional regressions?

File: src/components/Popover/index.native.js (hunk 1 above)
Comment (AlfredoAlc) [reply src/components/Popover/index.native.js:24]: I'll be working fixing this animation regressions

**Review by Jag96 — APPROVED** [review]
LGTM, thanks for the update!

**Pull request merged**